closure/templates/closure_js_template_library.bzl

```diff
@@ -66,6 +68,7 @@ _closure_js_template_library = rule(
         "plugin_modules": attr.label_list(),
         "should_provide_require_soy_namespaces": attr.bool(default = True),
         "should_generate_soy_msg_defs": attr.bool(),
+        "bidi_global_dir": attr.int(),
```

Comment:
attr.int(values = [1, -1])

Reply:
Cool, I didn't know about that. Updating

Reply:
Oh, if I do that, then it makes the default of None==0 invalid. I could provide a default value of 1 which is probably what most callers would want, but up to you.

Reply:
I guess making default one is good. As long as it doesn't break anything :)

Reply:
Done. The tests pass at least